scripts/standalone/index.js

```diff
@@ -31,27 +31,28 @@ const dev = (process.env.NODE_ENV !== 'production');
 const devPorts = dev || process.env.DEV_PORTS === 'true';
 
 const proxy = {
-  '/k8s':          proxyWsOpts(api), // Straight to a remote cluster (/k8s/clusters/<id>/)
-  '/pp':           proxyWsOpts(api), // For (epinio) standalone API
-  '/api':          proxyWsOpts(api), // Management k8s API
-  '/apis':         proxyWsOpts(api), // Management k8s API
-  '/v1':           proxyWsOpts(api), // Management Steve API
-  '/v3':           proxyWsOpts(api), // Rancher API
-  '/v3-public':    proxyOpts(api), // Rancher Unauthed API
-  '/api-ui':       proxyOpts(api), // Browser API UI
-  '/meta':         proxyMetaOpts(api), // Browser API UI
-  '/v1-*':         proxyOpts(api), // SAML, KDM, etc
+  '/k8s':            proxyWsOpts(api), // Straight to a remote cluster (/k8s/clusters/<id>/)
+  '/pp':             proxyWsOpts(api), // For (epinio) standalone API
+  '/api':            proxyWsOpts(api), // Management k8s API
+  '/apis':           proxyWsOpts(api), // Management k8s API
+  '/v1':             proxyWsOpts(api), // Management Steve API
+  '/v3':             proxyWsOpts(api), // Rancher API
+  '/v3-public':      proxyOpts(api), // Rancher Unauthed API
+  '/api-ui':         proxyOpts(api), // Browser API UI
+  '/meta':           proxyMetaOpts(api), // Browser API UI
+  '/rancherversion': proxyOpts(api), // Rancher version endpoint
+  '/v1-*':           proxyOpts(api), // SAML, KDM, etc
   // These are for Ember embedding
-  '/c/*/edit':     proxyOpts('https://127.0.0.1:8000'), // Can't proxy all of /c because that's used by Vue too
-  '/k/':           proxyOpts('https://127.0.0.1:8000'),
-  '/g/':           proxyOpts('https://127.0.0.1:8000'),
-  '/n/':           proxyOpts('https://127.0.0.1:8000'),
-  '/p/':           proxyOpts('https://127.0.0.1:8000'),
-  '/assets':       proxyOpts('https://127.0.0.1:8000'),
-  '/translations': proxyOpts('https://127.0.0.1:8000'),
-  '/engines-dist': proxyOpts('https://127.0.0.1:8000'),
+  '/c/*/edit':       proxyOpts('https://127.0.0.1:8000'), // Can't proxy all of /c because that's used by Vue too
+  '/k/':             proxyOpts('https://127.0.0.1:8000'),
+  '/g/':             proxyOpts('https://127.0.0.1:8000'),
+  '/n/':             proxyOpts('https://127.0.0.1:8000'),
+  '/p/':             proxyOpts('https://127.0.0.1:8000'),
+  '/assets':         proxyOpts('https://127.0.0.1:8000'),
+  '/translations':   proxyOpts('https://127.0.0.1:8000'),
+  '/engines-dist':   proxyOpts('https://127.0.0.1:8000'),
   // Plugin dev
-  '/verdaccio/':   proxyOpts('http://127.0.0.1:4873/-'),
+  '/verdaccio/':     proxyOpts('http://127.0.0.1:4873/-'),
```

Comment:
This needs rebasing

Reply:
It's already rebased. This is the standalone file. The changes in shell not applied here?